bin/build-package.js

```diff
@@ -0,0 +1,28 @@
+/** @format */
+
+// find all the packages
+const fs = require( 'fs' );
+const path = require( 'path' );
+const { execSync } = require( 'child_process' );
+
+const dir = process.cwd();
+const root = path.dirname( __dirname, '..' );
+
+const inputDir = path.join( dir, 'src' );
+const outputDirEsm = path.join( dir, 'dist', 'esm' );
+const outputDirCommon = path.join( dir, 'dist', 'cjs' );
+if ( fs.existsSync( inputDir ) ) {
+	console.log( 'Building %s', dir );
+	execSync( `node_modules/.bin/babel -d ${ outputDirEsm } ${ inputDir }`, {
```

Comment:
maybe move these to use `npx babel`?

Reply:
9abe31d5b3623060770ce55145c5c42c097efbfb